### PR TITLE
get rid of deprecated onActivityCreated in TimelineFragment

### PR DESCRIPTION
**Pull request by connyduck**

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt [hunk 1]
```diff
@@ -103,8 +103,6 @@ class TimelineFragment :
 
     private var isSwipeToRefreshEnabled = true
 
-    private var eventRegistered = false
```
[the remark this answers is not in the document]
Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt:106]: Is it though? Isn't it called when rotating screen etc?

[the remark this answers is not in the document]
Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt:106]: Wouldn't it keep fragments though? They get full lifecycle as well afaik

[the remark this answers is not in the document]
Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt:106]: I'll double check

[the remark this answers is not in the document]
Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt:106]: Ok, I checked. `onViewCreated` is never called twice on the same fragment without `onDestroy` in between, which cancels the subscribtion. (And while doing so I found a bug in `StatusListActivity`, which caused me to open #2332 😅 )

[the remark this answers is not in the document]
Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt:106]: Okay, that's what I would hope for, we are good then!

**Review by charlag — APPROVED** [review]

**Pull request merged**